ChatQnA/Dockerfile

```diff
@@ -1,31 +1,51 @@
-
-
 # Copyright (C) 2024 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-FROM python:3.11-slim
+ARG IMAGE_NAME=python
+ARG IMAGE_TAG=3.11-slim
+FROM ${IMAGE_NAME}:${IMAGE_TAG} as base
 
-RUN apt-get update -y && apt-get install -y --no-install-recommends --fix-missing \
+ENV LANG C.UTF-8
```

Comment:
Python base images already include this workaround for Python versions needing it:
* https://github.com/docker-library/python/blob/master/3.11/slim-bookworm/Dockerfile

(It's removed in 3.13 version.)

So I do not think such override to be needed for the base image.  Do you agree?